src/common/labelValuePair/labelValuePair.scss

```diff
@@ -0,0 +1,12 @@
+@import 'spacings';
+@import 'colours';
```

Comment:
If you are not using any color variable, no need to import this module.

Reply:
removed